workshops/meme-generator/README.md

```diff
@@ -0,0 +1,385 @@
+
+Now let's move on to the next part!
+
+![Excited doge](https://cloud-4560ajsyp.vercel.app/0giphy.gif)
+
+## Part 3: Building The Project
+
+### 1) HTML
+
+Let's first start by making a structure for our Meme Generator website.
+
+We'll be writing all our below code inside our `<body>` tag:
+
+* First give an appropriate heading to our website using the `<h1>` tag.
+
+* Next, want our memes to have two text fields: ***Top Text*** and ***Bottom Text***, something like this:
+
+    ![Meme showing top text and bottom text](https://cloud-ejplvs3dg.vercel.app/0image.png)
+
+    For that we'll require two text fields in our website, to achieve that we'll use `<textarea>` tag:
+
+    ```html
+    <textarea id="top-text"></textarea>
+    <textarea id="bottom-text"></textarea>
+    ```
+    
+    (put this after our `<h1>` tag)
+
+* Next, we'll need to take a meme template as a file input, we'll use:
+  
+  ```html
+  <input type="file" id="image-input" accept="image/*">
+  ```
+
+    This will pop-up a button on our website that'll let the user input a meme template as an image.
+
+* Now we need a generate button:
+
+    ```html
+    <button id="generate-btn">Generate!</button>
+    ```
+
+    Later we can add an event listener to this button which can run a function on click!
+
+* But we need something to display our meme, right?
+
+    For this, we'll use an [HTML5 Canvas](https://developer.mozilla.org/en-US/docs/Web/API/Canvas_API):
+
+    ```html
+    <canvas id="meme-canvas" title="Right click to save this meme"></canvas>
+    ```
+
+Our final HTML file will look like:
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <title>Meme Generator</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1>Welcome to Meme-Genie 🧞 </h1>
+        <div>
+            <textarea id="top-text"></textarea>
+            Text size: <input type="range" id="top-text-size-input" min="0.05" max="0.25" value="0.15" step="0.01">
+        </div>
+        <div>
+            <textarea id="bottom-text"></textarea>
+            Text size: <input type="range" id="bottom-text-size-input" min="0.05" max="0.25" value="0.15" step="0.01">
+        </div>
+        <div>
+            <input type="file" id="image-input" accept="image/*">
+        </div>
+        <div>
+            <button id="generate-btn">Generate!</button>
+        </div>
+        <div>
+            <canvas id="meme-canvas" title="Right click to save this meme"></canvas>
+        </div>
+        <script src="script.js"></script>
+    </body>
+</html>
+```
+
+Note that we have added the *Text Size* slider which we'll use to increase or decrease the text size.
+You can take a look at how different input types work [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input).
+
+### 2) CSS
+
+Next, let's add some basic styling to our website, navigate to our `style.css` file and add the following code:
```

Comment:
the comma is incorrect grammar here. should be a period followed by a new sentence